polaris-patterns/tests/setup/tests.ts

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable jest/require-top-level-describe */
+import {destroyAll} from '@shopify/react-testing';
+import {matchMedia} from '@shopify/jest-dom-mocks';
+import '@shopify/react-testing/matchers';
+import './matchers';
+
+beforeEach(() => {
+  matchMedia.mock();
```

Comment:
Moved the `matchMedia.mock()` to the test setup so devs don't have to add it every time they use AlphaCard